tests/basic_setters.rs

```diff
@@ -0,0 +1,38 @@
+#[macro_use] extern crate custom_derive;
+#[macro_use] extern crate derive_builder;
+
+custom_derive!{
+    #[derive(Debug, PartialEq, Default, Builder)]
+    struct Lorem {
+        ipsum: String,
+        dolor: String,
+        sit: i32,
+        amet: bool,
+    }
+}
+
+impl Lorem {
+    pub fn new<T: Into<String>>(value: T) -> Self {
```

Comment:
Oh, what a day, I didn't know multiple `impl Item`s are possible. Guess I never tried. :-)

But this would collide, if we reserved `fn new()` for ourselves.